canvas_sdk/commands/commands/prescribe.py

```diff
@@ -18,11 +18,11 @@ class Substitutions(Enum):
 
     fdb_code: str = Field(json_schema_extra={"commands_api_name": "prescribe"})
     icd10_codes: list[str] | None = Field(
-        None, json_schema_extra={"commandsd_api_name": "indications"}
+        None, json_schema_extra={"commands_api_name": "indications"}
```

Comment:
typo

Reply:
i'm surprised the command schema tests still passed with this typo... that worries me. i can look into it.

Reply:
ah its because we haven't enabled that test yet